example/basic/basic-model/pom.xml

```diff
@@ -67,6 +67,17 @@
             <artifactId>accumulo-minicluster</artifactId>
             <version>${accumulo.version}</version>
             <scope>test</scope>
+            <exclusions>
+                <!-- Conflicts with Reload4j -->
```

Comment:
Am I right in thinking that, not only will this stop dependency conflicts, but remove our log4j vulnerability - not only because Gaffer won't use it directly but it won't be using it indirectly through its dependencies?

If so, then will accumulo still be writing logs? And they will just be written by Reload4j rather than log4j?

Reply:
So far as I understand, that is correct. These changes cause log4j to be excluded from all transitive dependencies. Therefore the log4j JAR will not be on the classpath when any part of Gaffer is run. Any code in external dependencies (e.g. Accumulo) which makes calls to log4j or slf4j will instead be using Reload4j. Reload4j is a continuation of the log4j codebase and is compatible with code complied against log4j.

Reply:
I think it is key that it works with code that calls both slf4j AND log4j directly, because I think some of the dependencies (hdfs or Accumulo, I can't remember) still call log4j directly.

Reply:
Yes, a few of the dependencies use log4j directly and not through SLF4J at all. They all work as before. The dependencies don't know log4j is not used, the classfiles for reload4j have the same names I believe.